boxes/identity-server/confs/repository/deployment/server/eventpublishers/IsAnalytics-Publisher-wso2event-AuthenticationData.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
BTW, shall we decouple Configs from been burnt into the boxes and make them available with default configs ?
We can easily mount any deployment specific configs later when we are doing that specific deployment.

Reply:
+1